src/_about/faq.md

```diff
@@ -0,0 +1,142 @@
+---
+title: FAQ
+layout: about-page-faq
+description: Frequently asked questions, for the burning topics that need addressing regularly.
+
+redirect_from:
+- /faq/
+- /faq.html
+---
+
+# General Questions {#general-questions}
+
+{% admonition %}
+
+## What is Quilt?
+
+Quilt is a modding toolchain developed primarily for Minecraft that has been forked from The Fabric Project, primarily 
+due to concerns around governance, project management and moderation. You can read more about the Quilt project and 
+where it came from on [the About Us page](/about).
+
+{% endadmonition %}
+{% admonition %}
+
+## Which parts of Fabric is Quilt forking?
+
+All of them. Quilt is a hard fork of Fabric, and we aim to fork and maintain all projects necessary to exist
+independently of it.
+
+{% endadmonition %}
+{% admonition %}
+
+## Why should I use Quilt?
+
+For users, Quilt provides the following advantages:
```

Comment:
perhaps this should be "will provide"?

Reply:
No, I think it should be fairly obvious that it doesn't provide anything until it releases